Makefile

```diff
@@ -270,4 +270,3 @@ clean:
 
 documentation: $(SPHINX_BUILD) docs/*.rst
 	PYTHONPATH=src $(SPHINX_BUILD) -W -b html -d docs/_build/doctrees docs docs/_build/html
-	PYTHONPATH=src $(SPHINX_BUILD) -W -b doctest -d docs/_build/doctrees docs docs/_build/html
```

Comment:
Maybe consider putting this on a separate `make` task, rather than expunging it entirely? e.g.

```make
documentation-examples: documentation
	PYTHONPATH=src $(SPHINX_BUILD) -W -b doctest -d docs/_build/doctrees docs docs/_build/html
```

so that the task is still easy to run locally, it just doesn’t get run as part of CI?